purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesTransactionException.kt

```diff
@@ -0,0 +1,6 @@
+package com.revenuecat.purchases
+
+class PurchasesTransactionException(
```

Comment:
I wonder, could we mark this as `@JvmSynthetic` as well, since it's only going to be used in coroutines which are kotlin only? We didn't mark the `PurchasesException` though, so we might have to live with that...

Reply:
I think it's fine? Happy to mark it though

Reply:
Yeah, it's probably ok... We won't be using these exceptions in java-land for now, so I thought about limiting the API, but it should be ok since we need to keep the class for kotlin in any case.